tests/test_db_realistic_coverage.py

```
@@ -128,15 +144,21 @@ def test_database_error_recovery_scenarios(self):
                     # Should handle error gracefully
                 except Exception:
-                    pass
+                    logging.exception(
+                        "Unexpected exception in tests: test_db_realistic_coverage.py"
+                    )
 
+                    pass
             # Simulate permission errors
             with patch("sqlite3.connect", side_effect=PermissionError("Access denied")):
                 try:
                     conn = get_db_connection()
```

Ruff (0.13.3)

Local variable `conn` is assigned to but never used

Remove assignment to unused variable `conn`

(F841)